pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SingularEmailEventUpdates.java

```diff
@@ -0,0 +1,107 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.ddp.db.dao.EventActionSql;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.event.NotificationTemplate;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.List;
+
+@Slf4j
+public class SingularEmailEventUpdates implements CustomTask {
+
+    private static final String DATA_FILE  = "patches/update-email-delaySeconds.conf";
+    private static final String STUDY_GUID  = "singular";
+
+    protected Config dataCfg;
+    protected Path cfgPath;
+    protected Config cfg;
+    protected Config varsCfg;
+
+    @Override
+    public void init(Path cfgPath, Config studyCfg, Config varsCfg) {
+        if (!studyCfg.getString("study.guid").equals(STUDY_GUID)) {
```

Comment:
To avoid NPE I would invert the condition here:
`if (!STUDY_GUID.equals(studyCfg.getString("study.guid")))`